### PR TITLE
Multi zone guide refresh

### PR DESCRIPTION
**Pull request by jakubdyszkiewicz**
Refresh of https://github.com/kumahq/kuma-website/pull/402

### CODE REVIEW

File: docs/docs/1.4.1/deployments/how-multi-zone-works.md
Comment (lahabana) [inline docs/docs/1.4.1/deployments/how-multi-zone-works.md:3]: > Kuma supports running your service mesh in multiple zones. It is even possible to run with a mix of Kubernetes and Universal zones.
Probably shorter and simpler.

File: docs/docs/1.4.1/deployments/how-multi-zone-works.md
Comment (lahabana) [inline docs/docs/1.4.1/deployments/how-multi-zone-works.md:12]: I think it's missing a work in "If a zone consumes services only" Did you mean "If a zone consumers only services local to its zone"?

File: docs/docs/1.4.1/deployments/how-to-multi-zone.md
Comment (lahabana) [inline docs/docs/1.4.1/deployments/how-to-multi-zone.md:17]: Is CRD the right word? To me a CRD is the definition of a resource, instances of CRDs are just resources. So the reason we need them separated is to keep the global resources separated from the zone resources no?

File: docs/docs/1.4.1/deployments/how-to-multi-zone.md
Comment (lahabana) [inline docs/docs/1.4.1/deployments/how-to-multi-zone.md:34]: The 3685 appears here magically. you might want to reference the network page and mention that by default KDS runs on 3685 no?

File: docs/docs/1.4.1/deployments/how-to-multi-zone.md
Comment (lahabana) [inline docs/docs/1.4.1/deployments/how-to-multi-zone.md:213]: It might make sense to reference virtual-outbound to do more flexible things with hostnames and ports.

**Review by lahabana — COMMENTED** [review]
Some comments but it's a great improvement!

File: docs/docs/1.4.1/deployments/how-to-multi-zone.md
Comment (jakubdyszkiewicz) [reply docs/docs/1.4.1/deployments/how-to-multi-zone.md:34]: 5685 right? I added reference to ports.

File: docs/docs/1.4.1/deployments/how-multi-zone-works.md
Comment (jakubdyszkiewicz) [reply docs/docs/1.4.1/deployments/how-multi-zone-works.md:3]: While I agree it's shorter and simpler and the one above is wordy, those 2 sentences do not explain what the zone is.

File: docs/docs/1.4.1/deployments/how-multi-zone-works.md
Comment (jakubdyszkiewicz) [reply docs/docs/1.4.1/deployments/how-multi-zone-works.md:12]: Zone A can consumer services only from Zone A, but Zone B can also consume services from Zone A, so I'm not sure it's the same

Comment (Bradamant3) [reply docs/docs/1.4.1/deployments/how-multi-zone-works.md:12]: And neither option correctly explains zone ingress, right? (Yes I know I wrote the original. It's wrong.) I think we can trim to

```suggestion
* The **zone ingress** Provides ingress to services in different zones.
```

File: docs/docs/1.4.1/deployments/how-multi-zone-works.md
Comment (Bradamant3) [reply docs/docs/1.4.1/deployments/how-multi-zone-works.md:3]: Yeah, some of this belongs in a higher-level "about Kuma" topic, not here. Originally I was trying to trim and associate a bunch o' different bits of explanation from the current live doc. How about something like

```suggestion
Kuma supports the a service mesh that is distributed across multiple zones. Different zones in the same mesh can run on Kubernetes or in Universal mode, in different regions, on different cloud providers, or in different datacenters. The only thing that zones have in common is their connection to the same global control plane.
```

File: docs/docs/1.4.1/deployments/how-multi-zone-works.md
Comment (jakubdyszkiewicz) [reply docs/docs/1.4.1/deployments/how-multi-zone-works.md:12]: > Provides ingress to services in different zones

For me, it sounds like Ingress in zone A provides an entry point to services in zone B, C, D, which is not correct.

How about "Provides ingress to local services for different zones"

File: docs/docs/1.4.1/deployments/how-multi-zone-works.md
Comment (lahabana) [inline docs/docs/1.4.1/deployments/how-multi-zone-works.md:3]: I was suggesting only replacing the first 2 sentences not the whole paragraph.
```suggestion
Kuma supports running your service mesh in multiple zones. It is even possible to run with a mix of Kubernetes and Universal zones. Your mesh environment can include multiple isolated service meshes (multi-tenancy), and workloads running in different regions, on different clouds, or in different datacenters. A zone can be a Kubernetes cluster, a VPC, or any other deployment you need to include in the same distributed mesh environment.
```

File: docs/docs/1.4.1/deployments/how-multi-zone-works.md
Comment (lahabana) [reply docs/docs/1.4.1/deployments/how-multi-zone-works.md:12]: I like the way you reworded that!

File: docs/docs/1.4.1/deployments/how-multi-zone-works.md
Comment (lahabana) [inline docs/docs/1.4.1/deployments/how-multi-zone-works.md:30]: Use relative paths so that we stop getting bad versions :)

File: docs/docs/1.4.1/deployments/how-to-multi-zone.md
Comment (lahabana) [inline docs/docs/1.4.1/deployments/how-to-multi-zone.md:179]:
```suggestion
On Kubernetes, Kuma automatically tries to pick up the public address and port. Depending on your load balancing implementation, you might need to wait a few minutes for Kuma to get the address.
```

**Review by lahabana — APPROVED** [review]
LGTM though if I were I'd wait for #649 to be merged to get the link checker as this probably breaks some stuff.

**Pull request merged**
